.cirrus.yml

```diff
@@ -72,6 +72,14 @@ linux_task_template: &LINUX_TASK_TEMPLATE
     fingerprint_script:
       - echo "${CIRRUS_OS}"
       - echo "$(date +%Y).$(expr $(date +%U) / ${CACHE_PERIOD}):${CARTOPY_CACHE_BUILD}"
+    populate_script:
+      - conda create --quiet --name cartopy-cache cartopy
+      - source ${HOME}/miniconda/etc/profile.d/conda.sh >/dev/null 2>&1
+      - conda activate cartopy-cache >/dev/null 2>&1
+      - cd $(mktemp -d)
+      - wget https://raw.githubusercontent.com/bjlittle/cartopy/cartopy-feature-download/tools/cartopy_feature_download.py
```

Comment:
Temporary, for now... pending a PR to cartopy

Reply:
Nice!